RELEASE_NOTES.md

```diff
@@ -0,0 +1,225 @@
+# Chef Server Release Notes
+
+This document contains an overview of signficant customer-facing changes
+in the release. For a detailed list of changed components, refer to
+[CHANGELOG.md](CHANGELOG.md).
+
+This document contains release notes for the current release and all patches.
+For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
+
+## 12.3.0 (2015-11-12)
+
+### Chef Server
+  * `policy_name` and `policy_group` are now node attributes that will be
+    returned with the node object, and can be used in node searches.
+  * Search Improvements:
+    *  Elasticsearch is now supported as a search provider.  See 'Search'
+    * You can now provide your own Solr instance instead of using the one
+      bundled with chef-server.
+    * It is now possible for Chef Server to post updates directly to the
+      search provider for real-time search index updates, bypassing rabbitmq
```

Comment:
Not sure we should say real-time here.  There is still solr/elasticsearch indexing delay.